src/completion.ts

```diff
@@ -254,6 +254,10 @@ export async function isValidFunctionCompletionContext(filepath: string, positio
     }
 }
 
+function canCreateSnippetOfFunctionCall(kind: lsp.CompletionItemKind | undefined, options: WorkspaceConfigurationCompletionOptions): boolean {
+    return options.completeFunctionCalls === true && (kind === lsp.CompletionItemKind.Function || kind === lsp.CompletionItemKind.Method);
```

Comment:
```suggestion
    return options.completeFunctionCalls && (kind === lsp.CompletionItemKind.Function || kind === lsp.CompletionItemKind.Method);
```

Reply:
That would cause type error as `options.completeFunctionCalls` can be undefined and the function returns boolean.